OmsAgent/omsagent.py

```diff
@@ -68,12 +71,15 @@
 ExtensionStateSubdirectory = 'state'
 
 # Commands
```

Comment:
@nidhanda @abenbachir can I get a quick review of just this part; rest is signed off by Fikret

Reply:
Looks good to me.